### PR TITLE
feat(value) displaying html fields for mobile

### PR DESCRIPTION
**Pull request by tgadre**
## **Description**

Added a case for html/text area fields display. The css class is used in mobile to add custom styles
Also added a flag to strip out the html
screen shot 2018-03-09 at 3 53 53 pm

Novo will not be affected and is being tested

#### **Verify that...**

- [x] Any related demos where added and `npm start` still works
- [x] New demos work in `Safari`, `Chrome` and `Firefox`
- [x] `npm run lint` passes
- [ ] `npm test` passes and code coverage is increased
- [x] `npm run compile` still works

##### **Screenshots**

### CODE REVIEW

**Comment by jgodi** [comment]
|File (1)|Change (:bangbang: -25.75%)|
|---|---|
|Value.ts|:white_check_mark: +0.67%|

*Please note, individual file changes won't add up to the overall coverage due to number of lines!*

**All new files must be at 80% coverage and old files must NOT decrease in coverage!**

File: src/platform/elements/value/Value.spec.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { NovoValueElement, NOVO_VALUE_THEME, NOVO_VALUE_TYPE } from './Value';
 // import { NovoLabelService } from '../../services/novo-label-service';
 import { NovoValueModule } from './Value.module';
 // TODO fix specs
-describe('Elements: NovoValueElement', () => {
+fdescribe('Elements: NovoValueElement', () => {
```
Comment (amrutha-rajiv) [inline src/platform/elements/value/Value.spec.ts]: fdescribe

File: src/platform/elements/value/Value.ts [hunk 2]
```diff
@@ -96,6 +100,11 @@ export class NovoValueElement implements OnInit, OnChanges {
             }
         } else if (this.isEntityList(this.meta.type)) {
             this.type = NOVO_VALUE_TYPE.ENTITY_LIST;
+        } else if (this.isHTMLField(this.meta)) {
+            this.type = NOVO_VALUE_TYPE.HTML;
+            if (this.meta.stripHTML) {
+                this.data = this.data.replace(/<(.|\n)+?>/gi, '');
```
Comment (amrutha-rajiv) [inline src/platform/elements/value/Value.ts]: this can probably be done by the render pipe

File: src/platform/elements/value/Value.ts [hunk 3]
```diff
@@ -16,6 +16,10 @@ export enum NOVO_VALUE_THEME { DEFAULT, MOBILE };
                 <novo-entity-list *ngSwitchCase="NOVO_VALUE_TYPE.ENTITY_LIST" [data]='data' [meta]="meta"></novo-entity-list>
             </div>
 
+            <div *ngSwitchCase="NOVO_VALUE_TYPE.HTML" class="value-outer html-field">
+                <label>{{ meta.label }}</label>
```
Comment (amrutha-rajiv) [inline src/platform/elements/value/Value.ts]: if the class is the only thing that is different for `HTML` then we don't need a switchcase within the template - we can use some kind of variable to retrieve class value and use it in the default case

File: src/platform/elements/value/Value.spec.ts (hunk 1 above)
Comment (tgadre) [reply src/platform/elements/value/Value.spec.ts]: done

File: src/platform/elements/value/Value.ts (hunk 3 above)
Comment (tgadre) [reply src/platform/elements/value/Value.ts]: done

**Pull request merged**